tests/integration/interactive_init_test.go

```diff
@@ -81,6 +81,64 @@ var _ = Describe("odo init interactive command tests", func() {
 				Expect(helper.ListFilesInDir(commonVar.Context)).To(ContainElements("devfile.yaml"))
 			})
 
+			It("should allow for personalizing configurations", func() {
+				command := []string{"odo", "init"}
+				output, err := helper.RunInteractive(command, nil, func(ctx helper.InteractiveContext) {
+
+					helper.ExpectString(ctx, "Select language")
+					helper.SendLine(ctx, "Javascript")
+
+					helper.ExpectString(ctx, "Select project type")
+					helper.SendLine(ctx, "")
+
+					helper.ExpectString(ctx, "Select version")
+					helper.SendLine(ctx, "")
+
+					helper.ExpectString(ctx, "? Select container for which you want to change configuration?")
+					helper.SendLine(ctx, "runtime")
+
+					helper.ExpectString(ctx, "? What configuration do you want change?")
+					helper.SendLine(ctx, "Add new port")
+
+					helper.ExpectString(ctx, "? Enter port number:")
+					helper.SendLine(ctx, "3000")
```

Comment:
I would suggest adding a different port number since 3000 already exists in the devfile. 
```suggestion
					helper.SendLine(ctx, "5000")
```